File: semester 7/Metode Numerik/Kelompok/uts/2/2.py
hasil_pengukuran = 40
nilai_sebenarnya = 42
galat_mutlak = abs(hasil_pengukuran - nilai_sebenarnya)
galat_relatif = (galat_mutlak / nilai_sebenarnya) * 100
print("Galat Mutlak:", galat_mutlak)
print("Galat Relatif:", galat_relatif, "%")

File: semester 7/Metode Numerik/Kelompok/uts/2/2.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    double hasil_pengukuran = 40;
    double nilai_sebenarnya = 42;
    double galat_mutlak = fabs(hasil_pengukuran - nilai_sebenarnya);
    double galat_relatif = (galat_mutlak / nilai_sebenarnya) * 100;
    cout << "Galat Mutlak: " << galat_mutlak << endl;
    cout << "Galat Relatif: " << galat_relatif << "%" << endl;
    return 0;
}

